### PR TITLE
add WES import_vcf.py script

### PR DESCRIPTION
**Pull request by williamphu**
Adding our WES import_vcf.py script to a common repo.

### CODE REVIEW

**Comment by williamphu** [comment]
Thanks for the comments!

File: seqr_scripts/import_vcf.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    logger.info("Importing filters info...")
+    filters_ht = hl.import_vcf(
+        args.part_two_path,
+        reference_genome=args.import_build,
+        force_bgz=True,
+        find_replace=("nul", "."),
+    ).rows()
+    logger.info("Importing VCF...")
+    # Note: always assumes file is bgzipped
```
Comment (ch-kr) [inline seqr_scripts/import_vcf.py:33]:
```suggestion
    # NOTE: always assumes file is bgzipped
```
optional, I think most of the comments that start with note have "NOTE:" in the gnomad repo

**Review by ch-kr — DISMISSED** [review]
one tiny suggestion but LGTM!

**Comment by williamphu** [comment]
@ch-kr Sorry, could you approve again, looks like the suggested changes require new approval.

**Review by ch-kr — APPROVED** [review]

**Pull request merged**